### ISSUE TITLE
Terminal execute command from escaped brackets

### ISSUE THREAD

**Issue by jcubic** [jcubic/jquery.terminal#993]
### Issue summary

If you write XML inside escaped brackets, the terminal will invoke the HTML as extended command

### Expected behavior
It should print the brackets and apply formatting inside

### Steps to reproduce
1. Open [repro](https://codepen.io/jcubic/pen/YPKpjrp)

### Browser and OS
N/A

### Additional notes

The library probably unescapes the brackets on some point

**Comment by jcubic** [comment]
Another issue is calling:

```
term.echo($.terminal.escape_brackets($.terminal.apply_formatters(output)));
// Recursive loop in echo detected, skip
```

**Comment by iamazeem** [comment]
Hi @jcubic!
Thank you for creating this wonderful library! 🏆

I ran into a similar issue with JSON output:

```js
const term = $('body').terminal({});
const output = `[[[]]]`;
term.echo(output);
```

resulting into this error:

```text
Command '[' Not Found!
```

Setting the `raw` option fixed it:

```js
const term = $('body').terminal({});
const output = `[[[]]]`;
term.echo(output, { raw: true });
```

Output:

```json
[[[]]]
```

Shouldn't it be the default that the `echo` output is not treated as the new command? 🤔
Not sure if there are projects relying on this behavior.
However, providing an option to disable this would be great.

**Comment by jcubic** [comment]
It seems that escape_brackets doesn't work at all:

```javascript
const output = $.terminal.escape_brackets(`[[;white;]x]`);
term.echo(output);
// white x
```

It may be a regression that was not tested.

**Comment by iamazeem** [comment]
Thanks for the quick response! 👍 
Specifically for JSON, `$.terminal.escape_brackets` seems to be working fine.

For color codes too, it seems to be working fine on my side:

```
const output = $.terminal.escape_brackets(`[[;white;]x]`);
term.echo(output);
// [[;white;]x]
```

**Comment by jcubic** [comment]
Yes, it seems that the issue is only when xml formatter is included on the page. As in the description of the issue.

**Comment by jcubic** [comment]
The problem is the Unix formatting that was included. It calls `$.terminal.unescape_brackets` before processing.

I think that this can be closed. The users probably don't need both XML formatting and UNIX formatting.
